src/main/java/be/syntra/devshop/DevshopFront/services/ProductServiceImpl.java

```diff
@@ -76,13 +77,21 @@ public StatusNotification addProduct(@Valid ProductDto productDto) {
     }
```

Comment:
private final DataStore datastore is no longer necessary. Please remove

Reply:
removed